.github/workflows/deploy-docs.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.1
+        uses: actions/checkout@v3.3.1
```

```suggestion
        uses: actions/checkout@v3
```

I don't think `v3.3.1` is a real version :smile:  Tbh, we don't need to be pinned to a specific minor version here anyway.